config.toml.example

```diff
@@ -111,6 +111,28 @@
 # General build configuration options
 # =============================================================================
 [build]
+# The default stage to use for a compile.
+#
+# This is overridden by `doc-stage`, `build-stage`, etc.
+#stage = "none"
```

Comment:
I feel like this seems confusing. What will happen if users uncomment this which it becames `stage = "none"`? It will not work right?

Reply:
(see discussion in https://github.com/rust-lang/rust/pull/76625#discussion_r487371722)